.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -43,10 +43,10 @@ body:
       label: Python version
       description: What Python version?
       options:
+        - 3.13
         - 3.12
         - 3.11
-        - "3.10"
-        - 3.9
+        - 3.10
```

```suggestion
        - '3.10'
```
a string is needed here, otherwise it will be interpreted as a float